settings/config.settings.php

```diff
@@ -15,26 +15,33 @@
  * directory should be shared between all multi-sites, and each multisite will
  * override this selectively using configuration splits. However, some
  * applications may prefer to manage the configuration for each multisite
- * completely separately. If this is the case, they can set
- * $drs_override_config_directories to FALSE and
- * $settings['config_sync_directory'] = $dir . "/config/$site_dir" in
+ * completely separately. If this is the case,
+ * set FALSE to $drs_override_config_directories
+ * and $drs_override_site_studio_sync_directories
+ * $settings['config_sync_directory'] = $dir . "/config/$site_dir"
+ * $settings['site_studio_sync'] =  $dir . "/sitestudio/$site_dir" in
  * settings.php, and we will not overwrite it.
  */
-
+// phpcs:ignore
+$drs_override_config_directories = !isset($drs_override_config_directories) ? TRUE : FALSE;
+$drs_override_site_studio_sync_directories = !isset($drs_override_site_studio_sync_directories) ? TRUE : FALSE;
 
 /**
  * Site path.
  *
  * @var string $site_path
  * This is always set and exposed by the Drupal Kernel.
  */
- // phpcs:ignore
- $site_name = EnvironmentDetector::getSiteName($site_path);
+// phpcs:ignore
+$site_name = EnvironmentDetector::getSiteName($site_path);
+
 // phpcs:ignore
 // Config sync settings.
-$settings['config_sync_directory'] = "../config/" . $site_name;
+$settings['config_sync_directory'] = $drs_override_config_directories ?
+"../config/$site_name" : "../config/default";
 // Site Studio sync settings.
-$settings['site_studio_sync'] = "../sitestudio/" . $site_name;
+$settings['site_studio_sync'] = $drs_override_site_studio_sync_directories ?
+"../sitestudio/$site_name" : "../sitestudio/default";
```

Comment:
As per my knowledge with Ternary Operator first is true then false. Here if the $drs_override_site_studio_sync_directories is true then config will be exported in "../sitestudio/default" else in "../sitestudio/$site_name", right?
```suggestion
$settings['site_studio_sync'] = $drs_override_site_studio_sync_directories ?
"../sitestudio/default" : "../sitestudio/$site_name";
```